content/en/infrastructure/resource_catalog/policies/_index.md

```diff
@@ -30,30 +35,43 @@ Specifically, you can:
 
 ## Create a custom policy
 
-{{< img src="/infrastructure/resource_catalog/governance/custom-policy-example-1.png" alt="A custom policy reflecting a compliance score of thirty percent." width="100%">}}
+{{< img src="/infrastructure/resource_catalog/policies/custom-policy-example-2.png" alt="A custom policy reflecting a compliance score." width="100%">}}
 
 Custom policies require specific values in your cloud resource attributes within Datadog based on your organization's infrastructure best practices.
 
 To create a custom policy:
 
-1. Navigate to **Infrastructure > Resource Catalog** and click the [**Governance**][1] tab.
+1. Navigate to **Infrastructure > Resource Catalog** and click the [**Policies**][1] tab.
 2. Click the **New Custom Policy** button.
 3. Select a resource type from the dropdown menu.
-4. Optionally, search for additional dataset filters.
+4. Optionally, search for additional dataset filters, such as `env: prod` to only include resources in production.
 5. Select a target resource attribute and desired value.
 6. Optionally, add instructions for remediation.
 7. A name is automatically generated based on the data entered, but you can modify it.
 8. Click **Create Custom Policy**.
 
 Click the new policy to review all non-compliant resources and filter them by region, environment, account, service, or team. You can also group them by attributes or tags.
 
+### Selecting values for your target attribute
+
+Custom policies let you define a target resource attribute and a desired value, providing flexibility in creating policies for your cloud resources without requiring complex query languages. The following features are available:
+
+* **Access data in nested attributes**:
+Validate more of your configurations (for example mandate that `TLS 1.2` stored in a nested property is used for Amazon CloudFront). 
+* **Use advanced condition matching**:
+Use operators like `\>, \<, \!=` (for example enforcing Kubernetes version \> 1.25).  
```

Comment:
```suggestion
Use operators like `\>`, `\<`, or `\!=` (for example, enforcing Kubernetes version \> 1.25).  
```

Reply:
It took me a moment to realize that these were three separate operators, so suggesting this update to clarify